api_spec.yaml

```diff
@@ -669,15 +675,17 @@ paths:
                     properties:
                       status:
                         $ref: '#/components/schemas/statusOK'
-                      newScopes:
-                        type: array
-                        items:
-                          $ref: '#/components/schemas/oauth2Scope'
                   - type: object
                     properties:
                       status:
                         type: string
-                        enum: ['UNKNOWN_CLIENT_ID']
+                        enum: ['UNKNOWN_CLIENT_ID_ERROR']
+                  - type: object
+                    properties:
+                      status:
+                        type: string
+                        enum: ['UNKNOWN_SCOPE_ERROR']
+
```

Comment:
Why does this PR have changes for delete scope for client API?

Also, I just realised, we are using UNKNOWN_SCOPE_ERROR with many meanings. In some APIs, it means that the scope doesn't exist in the global scopes table. Whereas here it means that the client did not have the scope.

We should not change the meaning of the error codes this way. Remove the UNKNOWN_SCOPE_ERROR from this API, and instead, add a boolean like `didScopeExist` in the status: OK response of the delete API

Reply:
I think some commits might have missed while the previous PR was merged. Now the issue is resolved.

Reply:
Changes made on delete client scope api response

Reply:
The flow diagram is not synced with these changes..

Reply:
Checked again.. Changes are available
https://app.code2flow.com/flowcharts/6478d1783f4c2482c82c7b5a